constants.js

```diff
@@ -13,6 +13,12 @@ export const rarityColors = {
   legendary: [0xAD00EA, 0x32002D],
 };
 
+export const PEEK_FACE_INDICES = [
```

Comment:
IMO this should be dynamically computed to make it clear how we arrived at this constant.